.travis.yml

```diff
@@ -41,15 +38,18 @@ jobs:
           stage: Test
           after_success: beaver dlang codecov
           script:
+              - set -e
+              # can't set it in env section or it will be overwritten by actual
+              # test definition after anchor is injected
```

Comment:
I don't fully understand this comment and why setting a global and overriding it where needed doesn't work.

Reply:
Because overriding does not append - I would need to repeat all globally defined env vars again. Maybe you know of a better way?